.golangci.yml

```diff
@@ -5,72 +5,43 @@ run:
   skip-dirs:
     - pkg/pb
 linters:
-  enable-all: true
+  enable:
```

Comment:
Why did you change `enable-all`? It should stay as-is and we just disable the linters we are not interested in.